taesch/lib/api/map_api_logic/api_querier.dart

```diff
@@ -7,48 +7,72 @@ import 'package:http/http.dart';
 import 'package:taesch/api/map_api_logic/overpass_query_indexes.dart';
 import 'package:taesch/api/repository.dart';
 import 'package:taesch/exceptions/custom/query_exceptions.dart';
+import 'package:taesch/middleware/log/log_level.dart';
+import 'package:taesch/middleware/log/logger_wrapper.dart';
+import 'package:taesch/model/log_message.dart';
 import 'package:taesch/model/map_spot.dart';
+
+
 // import 'package:taesch/utils/my_tools.dart';
 
 // import 'geolocation_tools.dart';
 // import 'querying_tools.dart';
 
 class APIQuerier {
```

Comment:
Möglicherweise sollte die Klasse in QuerierAPI umbenannt werden -> APIQuerier hört sich an, als würde nach einer API gesucht werden

Reply:
wurde in Refactoring gemacht